test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -39,13 +40,11 @@ public class BasicTests
             var server = TestHelper.CreateServer(_app, SiteName, _configureServices);
             var client = server.CreateClient();
             var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
-
-            // The K runtime compiles every file under compiler/resources as a resource at runtime with the same name
-            // as the file name, in order to update a baseline you just need to change the file in that folder.
-            var expectedContent = await _resourcesAssembly.ReadResourceAsStringAsync("compiler/resources/BasicWebSite.Home.Index.html");
+            var outputFile = "compiler/resources/BasicWebSite.Home.Index.html";
+            var expectedContent =
```

Comment:
Why duplicate the code everywhere when you can just have a single helper method of doing it though? Makes writing the tests fool proof, aka you get updateable resource files for free.